### ISSUE TITLE
[Bug] Infinite Git credentials requests

### ISSUE THREAD

**Issue by enricobenedos** [ForkIssues/TrackerWin#764]
Good morning,
I've changed my git password and after trying to push my commits Fork shows me a lot of Git Manager Credentials windows
![image](https://user-images.githubusercontent.com/12233746/80380175-7c9c0a00-889f-11ea-89e6-7d8464092de4.png)

I'm using Fork `1.48.2.0`

**Comment by DanPristupov** [comment]
I suppose it's not push, but automatic fetch which happened to run at the same time.

Open Credential Manager in Windows Start menu, remove the entry related to your server and fetch/push again.

**Comment by enricobenedos** [comment]
At the moment I simply solved closing and reopening Fork and immediately complete the password field. I think your suggestion will sure fix the problem. 
I opened this issue because seems that Fork does not await for WindowsSecurity popup to complete the authentication but simply recall it causing a lot of opened windows. If it is an expected behaviour it's not a problem for me, it is only strange.
